File: src/main/java/dev/thornium/flatcore/common/data/FTItems.java
package dev.thornium.flatcore.common.data;

import com.tterrag.registrate.util.entry.ItemEntry;
import net.minecraft.world.item.Item;

import static dev.thornium.flatcore.FlatCore.REGISTRATE;

public class FTItems {
    static {
        REGISTRATE.creativeModeTab(() -> FTCreativeTabs.FLATCORE);
    }

    public static ItemEntry<Item> GALLIUM_BOULE = REGISTRATE.item("gallium_iii_oxide_boule", Item::new)
            .lang("Gallium III Oxide Boule")
            .register();
    public static ItemEntry<Item> GALLIUM_WAFER = REGISTRATE.item("gallium_iii_oxide_wafer", Item::new)
            .lang("Gallium III Oxide Wafer")
            .register();

    public static void init() {}
}

File: src/main/kotlin/dev/thornium/flatcore/common/data/FTItems.kt
package dev.thornium.flatcore.common.data

import com.tterrag.registrate.util.entry.ItemEntry
import dev.thornium.flatcore.FlatCore.Companion.REGISTRATE
import dev.thornium.flatcore.api.Initialized
import net.minecraft.world.item.Item

object FTItems : Initialized {
    init {
        REGISTRATE.creativeModeTab { FTCreativeTabs.FLATCORE }
    }

    val GALLIUM_BOULE: ItemEntry<Item> = REGISTRATE.item("gallium_iii_oxide_boule", ::Item)
        .lang("Gallium III Oxide Boule")
        .register()

    val GALLIUM_WAFER: ItemEntry<Item> = REGISTRATE.item("gallium_iii_oxide_wafer", ::Item)
        .lang("Gallium III Oxide Wafer")
        .register()
}
